spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/json-shade/java/org/springframework/boot/configurationprocessor/json/JSONStringer.java

```diff
@@ -50,7 +50,7 @@
  * create more than 20 levels of nesting may fail with a {@link JSONException}.
  * <p>
  * Each stringer may be used to encode a single top level value. Instances of this class
- * are not thread safe. Although this class is nonfinal, it was not designed for
+ * are not thread safe. Although this class is non-final, it was not designed for
```

Comment:
This class is "shaded" from third-party source and should be left as-is.

Reply:
Done.